tests/tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = {ocs-tests}
+basepython = py37
+skipsdist = true
+
+[testenv]
+deps=
+  -rrequirements.txt
+  pytest
```

Comment:
I see pytest is defined just here in tox requirements. But if we are going support pytest in OCS-CI, we will be implementing some plugins and so on I think that it should be requirements defined for whole ocs-ci project in setup.py.

Reply:
Good point, I'll make that change.